pkg/logproto/bloomgateway.proto

```diff
@@ -45,6 +45,7 @@ message GroupedChunkRefs {
   uint64 fingerprint = 1;
   string tenant = 2;
   repeated ShortRef refs = 3;
+  IndexSeries labels = 4;
```

Comment:
nit: GroupChunkRefs is shared between the request and the response, but the response never provides it; can we either:

* Find a way so the labels can only be provided by the request, or 
* Comment here that `labels` is an optional field and may not be set? 

(I'm fine with either)